.k8s-helpers

```diff
@@ -174,12 +174,12 @@ kpfp() {
   local -r pod="${1:-$(kubectl get pod --no-headers | choose 0 | fzf)}"
   local -r data="$(kubectl get pod "${pod}" -ojson)"
   if [[ "$(jq '.spec.containers | length' <<< "${data}")" = 1 ]]; then
-    container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
+    local -r container="$(jq ".spec.containers[0].name" -r <<< "${data}")"
   else
-    container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
+    local -r container="$(jq ".spec.containers[].name" -r <<< "${data}" | fzf --header='mutliple containers; please pick one')" # user choice
   fi
   datajson="$(jq ".spec.containers[] | select(.name==\"${container}\")" <<< "${data}")"
-  if [[ "$(jq ".ports" <<< "${data}")" == "null" ]]; then
+  if [[ "$(jq ".ports" <<< "${datajson}")" == "null" ]]; then
```

Comment:
i think this is the main bit, we need to get the port from `datajson` and not `data`
not sure how this worked with `data` 🤔

Reply:
for posterity, we later discussed that this function was not used that much coz we all used `kpfs`